Makefile

```diff
@@ -73,16 +73,16 @@ export TEST_ASSET_KUBE_APISERVER := $(KUBEBUILDER)/kube-apiserver
 export TEST_ASSET_ETCD := $(KUBEBUILDER)/etcd
 
 vendor:
-	@ $(ECHO) "\033[36mPuling dependencies\033[0m"
-	$(DEP) ensure --vendor-only
+	@ $(ECHO) "\033[36mPuling dependencies\033[0m\n"
+	go get -v github.com/wave-k8s/wave/cmd/manager
```

Comment:
`go mod download -x`?

Reply:
ohh right, I just tested and noticed that the Makefile's need some love so I'm working on them.. will take some testing but I hope to get generation +kustomize and as much as I can validated